tools/helixer/helixer.xml

```diff
@@ -68,7 +67,7 @@
             </sanitizer>
             <validator type="regex">[0-9a-zA-Z_]+</validator>
         </param>
-
+        <param name="size" argument="--batch-size" type="integer" value="8" label="Batch size" help="It may then be necessary to reduce it if the GPU runs out of memory" />
```

Comment:
```suggestion
        <param name="size" argument="--batch-size" type="integer" value="8" label="Batch size" help="It may be necessary to reduce it if the GPU runs out of memory" />
```